### ISSUE TITLE
Improve `explicit-length-check`?

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#875]
Pass

```js
if (foo.length === 0) {}
```

Fail

```js
if (foo.length !== 0) {}
```

These two looks the same thing to me, but one fail one pass. Seems weird

Do you think we should revert #850, and find a better solution?

**Comment by fisker** [comment]
Make sense.
